graphql/http_handler.go

```diff
@@ -15,6 +16,8 @@ import (
 	"github.com/vektah/gqlparser/v2/gqlerror"
 )
 
+const ContextCanceledErrorMessage = "context canceled"
```

Comment:
Rather than duplicate the `context.Canceled` error, it seems like you could just use `context.Canceled.Error()`.